content/en/docs/configuration/_index.md

```diff
@@ -1,23 +1,22 @@
 ---
-title: "Configuration"
+title: "Issuer Configuration"
 linkTitle: "Configuration"
 weight: 30
 type: "docs"
 ---
 
-In order to configure cert-manager to begin issuing certificates, first
-`Issuer` or `ClusterIssuer` resources must be created. These resources represent
-a particular signing authority and detail how the certificate requests are going
-to be honored. You can read more on the concept of `Issuers`
-[here](../concepts/issuer/).
+The first thing you'll need to configure after you've installed cert-manager will be an issuer
+which will be used for signing your certificates.
 
-cert-manager supports multiple 'in-tree' issuer types that are denoted by being
-in the `cert-manager.io` group. cert-manager also supports external issuers than
-can be installed into your cluster that belong to other groups. These external
-issuer types behave no different and are treated equal to in tree issuer types.
+This section documents how the different issuer types can be configured. You might want to
+read more about `Issuer` and `ClusterIssuer` resources [here](../concepts/issuer/).
 
-When using `ClusterIssuer` resource types, ensure you understand the [`Cluster
-Resource Namespace`](../faq/cluster-resource/) where other Kubernetes resources
-will be referenced from.
+cert-manager supports 'in-tree' issuer types - denoted by being in the `cert-manager.io`
+group - as well as external issuers which can be installed in addition to the in-tree types.
+Both in-tree and external issuers are treated the same and are configured similarly.
```

Comment:
Suggestion:
```suggestion
cert-manager comes with a number of built-in certificate issuers - denoted by being in the `cert-manager.io`
group. Additionally external issuers can be installed in addition to the built-in ones.
Both built-in and external issuers are treated the same and are configured similarly.
```
.. or something such with the idea to just communicate the fact that there are built-in issuers + external issuers are a thing too.

Reply:
This reads a lot better as "built-in" for sure. I'm not sure why I didn't drop use of "in-tree" originally! Thanks :grin: